auth_backends/backends.py

```diff
@@ -99,6 +103,169 @@ def _map_user_details(self, response):
         return dest
 
 
+class EdXSettingsAwareOpenIdConnect(OpenIdConnectAuth):
```

Comment:
Why not inherit from `EdXOpenIdConnect`? Doing so would get rid of a lot of duplicated code.